### PR TITLE
Integrate and document backup repository

### PR DESCRIPTION
**Pull request by yetinam**
- make remote root configurable in config
- add a backup repository
- add a function to switch to the backup repository
- document new functionality in known issues and documentation

### CODE REVIEW

File: seisbench/__init__.py [hunk 1]
```diff
@@ -52,3 +47,33 @@
...
+    """
+    Use the backup repository instead of the original one. This is helpful if the main repository is unavailable
+    or if your institution/provider blocks access to the main repository. However, this might lead to degraded
+    download speeds.
+
+    :param backup: URL of the backup repository. The default is hard-coded.
+    """
+    logger.warning(
+        f"Setting remote root to: {backup}\n"
+        f"Please note that this can affect you download speed."
```
Comment (pevans-gfz) [inline seisbench/__init__.py:73]: you -> your

File: README.md [hunk 2]
```diff
@@ -75,6 +75,12 @@ Check out the [contribution guidelines](https://github.com/seisbench/seisbench/b
 
 ## Known issues
 
+- Some institutions and internet providers are blocking access to our data and model repository, as it is running on a non-standard port (2880).
+  This usually manifests in timeouts when trying to download data or model weights.
+  To verify the issue, try accessing [https://hifis-storage.desy.de:2880/](https://hifis-storage.desy.de:2880/) directly from the same machine.
+  As a mitigation, you can use our backup repository. Just run `seisbench.use_backup_repository()`.
+  Please note that the backup repository will usually show lower download speeds.
+  We recommend contacting your network administrator to allow access to port 2880 on our server as a more performant solution.
```
Comment (pevans-gfz) [inline README.md:83]: Suggestion: "allow outgoing access" to port 2880.

"more performant" -> "higher performance".

Comment (pevans-gfz) [reply README.md:83]: Also, "allow outgoing access to TCP port 2880". (Your network administrator will be happier opening a smaller hole. :)

**Comment by yetinam** [comment]
Thanks @pevans-gfz , I've incorporated the suggestions.

**Pull request merged**